sites/public/pages/finder.tsx

```diff
@@ -152,14 +176,21 @@ const Finder = () => {
                             key={FrontendListingFilterStateKeys[field.label]}
                           >
                             <Field
-                              name="bedRoomSize"
+                              name={activeQuestion.fieldGroupName}
                               register={register}
                               id={FrontendListingFilterStateKeys[field.label]}
-                              label={t(`listingFilters.bedroomsOptions.${field.translation}`)}
+                              label={
+                                field.translation
```

Comment:
What are the cases where we don't want the translation?

Reply:
@emilyjablonski I included this to account for the names of the Regions.